aten/src/ATen/native/quantized/cpu/fbgemm_utils.cpp

```diff
@@ -109,18 +109,12 @@ fbgemm::conv_param_t<kSpatialDim> MakeFbgemmConvParam(
     const std::vector<int>& dilations,
     const std::vector<int>& output_padding,
     bool transposed) {
-  // NOLINTNEXTLINE(cppcoreguidelines-pro-type-member-init)
-  std::array<int, kSpatialDim> image_shape_;
-  // NOLINTNEXTLINE(cppcoreguidelines-pro-type-member-init)
-  std::array<int, kSpatialDim> kernels_;
-  // NOLINTNEXTLINE(cppcoreguidelines-pro-type-member-init)
-  std::array<int, kSpatialDim> strides_;
-  // NOLINTNEXTLINE(cppcoreguidelines-pro-type-member-init)
-  std::array<int, kSpatialDim * 2> pads_;
-  // NOLINTNEXTLINE(cppcoreguidelines-pro-type-member-init)
-  std::array<int, kSpatialDim> dilations_;
-  // NOLINTNEXTLINE(cppcoreguidelines-pro-type-member-init)
-  std::array<int, kSpatialDim> output_padding_;
+  std::array<int, kSpatialDim> image_shape_{};
```

Comment:
All these arrays are immediately populated right below. That being said, it would probably make sense to construct them directly from the std::array using move constructors. Also does std::move below actually work with const ref values provided?!

Reply:
No such constructor in std::array, but we can add an overload accepting std::initialization_list.